app/assets/js/ui/card/card.scss

```diff
@@ -58,7 +58,7 @@ $root: $eps-prefix + card;
 		object-position: top;
 		position: absolute;
 		top: 0;
-		left: 0;
```

Comment:
Please review it in the front end

Reply:
Tested. Works as expected.

This change is for consistency:
- All `left: 0;` properties replaced with `@include start(0);`
- All `right: 0;` properties replaced with `@include end(0);`

We do the same in other places. In the App I found a few places where LRT properties have been hardcoded.

Reply:
@arielk 

In this specific case, it doesn't matter if we use `left` or `right` because we have `width: 100%;` and `position: absolute;`. In both cases it will cover 100% no matter where is the starting point.